tests/linux_kernel/test_threads.py

```diff
@@ -50,3 +50,8 @@ def test_crashed_thread(self):
             "crashed thread is only defined for core dumps",
             self.prog.crashed_thread,
         )
+
+    def test_thread_name(self):
+        pid = os.getpid()
+        thread = self.prog.thread(pid)
+        self.assertEqual(thread.object.comm.string_(), thread.name)
```

Comment:
Testing against `thread.object.comm.string_()` is almost a tautology since that's how this works under the hood. Let's test against reading `/proc/{os.getpid()}/comm` instead.